src/lexer.coffee

```diff
@@ -46,6 +46,7 @@ exports.Lexer = class Lexer
     @seenImport = no             # Used to recognize IMPORT FROM? AS? tokens.
     @seenExport = no             # Used to recognize EXPORT FROM? AS? tokens.
     @exportSpecifierList = no    # Used to identify when in an EXPORT {...} FROM? ...
+    @importSpecifierList = no    # Used to identify when in an IMPORT {...} FROM? ...
```

Comment:
Please keep it consistent and always put the import stuff before the export stuff.